src/__tests__/device-test.ts

```diff
@@ -469,3 +469,24 @@ test('getInstallationId', async () => {
         installationId: '123',
     });
 });
+
+test('getAppDomain', async () => {
+    createFakeWebKitPostMessage({
+        checkMessage: (msg) => {
+            expect(msg.type).toBe('GET_APP_DOMAIN');
+        },
+        getResponse: (msg) => ({
+            type: 'GET_APP_DOMAIN',
+            id: msg.id,
+            payload: {
+                appDomain: '123',
```

Comment:
If we expect an URL IMHO I would use https://example.com if there is any string user `ANY_STRING`

```suggestion
                appDomain: 'https://example.com',
```